cyto_dl/models/im2im/diffusion_autoencoder.py

```diff
@@ -170,13 +170,19 @@ def forward(self, x_cond, x_diff):
     def _generate_image(self, noise, cond):
         self.scheduler.set_timesteps(num_inference_steps=self.hparams.n_inference_steps)
         with torch.no_grad():
-            sample = self.inferer.sample(
+            sample, intermediates = self.inferer.sample(
                 input_noise=noise,
                 diffusion_model=self.autoencoder,
                 scheduler=self.scheduler,
+                save_intermediates=True,
                 conditioning=cond,
                 verbose=False,
+                intermediate_steps=1,
             )
+        # during training, final image is all-nan, try to return last non-nan image
```

Comment:
why does this happen?

Reply:
It's a (known, I think) bug with the linear beta scheduler